check/python_module.go

```diff
@@ -51,6 +53,17 @@ func (c *pythonModuleVersion) validate() error {
 			c.Relationship, c.ID())
 	}
 
+	switch c.MinRelationship {
+	case "":
+		grip.Debug("no relationship specified, using greater than or equal to (gte)")
+		c.MinRelationship = "gte"
+	case "gte", "lte", "lt", "gt", "eq":
+		grip.Debugln("relationship for '%s' check set to '%s'", c.ID(), c.MinRelationship)
+	default:
+		return errors.Errorf("relationship '%s' for check '%s' is invalid",
+			c.MinRelationship, c.ID())
+	}
+
```

Comment:
this is mostly redundant to the implementation of compareVersion (you could add the debug logging for "" there, if you really want.)